modules/dxp/apps/analytics/analytics-reports-journal-impl/src/main/java/com/liferay/analytics/reports/journal/internal/info/item/JournalArticleAnalyticsReportsInfoItem.java

```diff
@@ -219,32 +311,26 @@ private Layout _getLayout(JournalArticle journalArticle) {
 				return null;
```

Comment:
This might be Optional.empty(), right?

Reply:
It is Optional.empty(), I don't understand the question, sorry :(

Reply:
You are reviewing the code of the old version maybe?

Reply:
ohh, seriously? let me checkout it again, it was in `_getLayoutOptional` method L311

Reply:
It still exists, but It comes from a previous commit. The method should return an Optional and not a null, right?

Reply:
Yeahh!!!, it should return an optional